### ISSUE TITLE
USER ISSUE: Can't start a new world

### ISSUE THREAD

**Issue by TopSecretPorkChop** [StrangeLoopGames/EcoIssues#23889]
\
**User Id: (slg40279)**
**Version:** 0.10.0.0 beta playtest-39

**Steps to Reproduce:**
Create a new local world (last option in the New Game menu)
Leave all settings as default except "Generate new terrain"

**Expected behavior:**
Terrain should generate, world should be created, and the map screen is brought up for selecting start location.

**Actual behavior:**
The following error message popped up:
```
<size=150%>Connection Failed</size>

Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Items.Item.GetItemByString(User user, String item)
   at Eco.Gameplay.Ite</size>...

Check log file to see full message.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by TopSecretPorkChop** [comment]
It seems this issue may be related to the new avatar system. I clicked on the Avatar button on the main menu. Though my avatar settings seem to still have been remembered, when I finished that process and got back to the main menu, there was a picture of my avatar which wasn't there before (maybe it had been removed from local cache or something when I reinstalled trying to fix another issue recently). Anyway, that seems to have fixed the problem as I was able to log into a local world (after a hang-up on Loading Items after the Connecting message).
